app/Http/Controllers/Auth/EmailChangeController.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use Illuminate\Http\Request;
+use App\Notifications\ConfirmEmail;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Database\Eloquent\Model;
+use App\Http\Requests\EmailChangeRequest;
+
+class EmailChangeController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @param  \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        if ($request->session()->has('user_id')) {
+            $model = config('auth.providers.users.model');
```

Comment:
I understand why you've done this, but in our case, I don't think we'll ever change the default `users.model` in an installation... so why do this?

Reply:
Indeed. It's a copy/paste